src/Wallabag/CoreBundle/Helper/ContentProxy.php

```diff
@@ -184,7 +184,8 @@ private function stockEntry(Entry $entry, array $content)
         }
 
         if (!empty($content['date'])) {
-            $this->updatePublishedAt($entry, $content['date']);
+            $contentDate = $content['date'] instanceof \DateTime ? $content['date']->date : $content['date'];
+            $this->updatePublishedAt($entry, $contentDate);
```

Comment:
Yeah this is a good start but you should add that condition in `updatePublishedAt` instead